static/app/views/starfish/utils/useSpansQuery.tsx

```diff
@@ -101,8 +103,10 @@ export function useWrappedDiscoverTimeseriesQuery({
   enabled,
   initialData,
   referrer,
+  cursor,
```

Comment:
Hmm, do timeseries queries use cursors? Either way, I don't think this breaks anything